suites/reef/rbd/tier-2_rbd_rest.yaml

```diff
@@ -0,0 +1,100 @@
+# Tier2: Usecase to cover RBD pool and image creation using REST API
+# Tier-level: 2
+# Test-Suite: tier-2_rbd_rest.yaml
+#
+# Ceph RBD component.
+#
+# Cluster Configuration:
+#    Node 4 must to be a client node
+tests:
+
+  # Setup the cluster
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
```

Comment:
lets not skip monitoring stack

Reply:
yeah, this is just a POC of REST framework, yaml specific is part of testcase writing